BookingProject/BookingWebAPI/BookingWebApi.Application/Services/TokenService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using BookingWebApi.Application.Interfaces;
+using BookingWebApi.Domain.Entities;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.Services
+{
+    public class TokenService : ITokenService
+    {
+        private readonly IConfiguration _config;
```

Comment:
We don't inject `IConfiguration` directly, but use `IOptions` pattern. Please refactor